src/content/en/fundamentals/instant-and-offline/get-started/httprequests-5.md

```diff
@@ -0,0 +1,168 @@
+﻿project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-10-18 #}
+{# wf_published_on: 2017-10-18 #}
+
+# HTTP Requests {: .page-title }
+
+{% include "web/_shared/contributors/dgash.html" %}
+
+
+Everything a web page needs to be a web page -- text, graphics, styles, scripts, everything -- must be downloaded from a server via an HTTP request. As much as 80% of a page's display time is spent downloading those components (source: [Yahoo!](https://developer.yahoo.com/performance/rules.html#num_http)). So far, we've talked about reducing the size of those downloads by compressing images, minifying CSS and JavaScript, zipping the files, and so on. But there's a more fundamental approach: in addition to just reducing download *size*, let's also consider reducing download *frequency*.
```

Comment:
I would try to keep the statistics we reference within the last ~ 5 years where possible. The above Yahoo performance page was published in 2006 and should be dropped or replaced. In general, sites like HTTPArchive or WPOStats are good to take a look at when we're looking for up to date numbers for quotes like this.